app/views/users/show.html.erb

```diff
@@ -1,3 +1,4 @@
 <h1> User <%= @user.full_name %> </h1>
 <h2 class="user-role"> <%= @user.role %> </h2>
+<h2 class="user-intent"> <%= @user.intent %> </h2>
```

Comment:
Actually, perhaps the better solution is adding a third "undeclared" value to the enum that it defaults to?